pkgs/applications/networking/instant-messengers/teamspeak/client.nix

```diff
@@ -38,39 +83,36 @@ stdenv.mkDerivation rec {
   };
 
   # grab the plugin sdk for the desktop icon
-  pluginsdk = fetchurl {
-    url = "http://dl.4players.de/ts/client/pluginsdk/pluginsdk_3.1.1.1.zip";
-    sha256 = "1bywmdj54glzd0kffvr27r84n4dsd0pskkbmh59mllbxvj0qwy7f";
+  pluginsdk = fetchzip {
```

Comment:
Was there a reason not to use fetchzip? With unzip (as before) the file is extracted into a directory named `ts_sdk_3.3.1` which makes it a little more difficult.

Reply:
None. I always prefer the most specific fetcher too.